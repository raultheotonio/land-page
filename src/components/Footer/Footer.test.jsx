import { screen } from '@testing-library/react';
import { Footer } from '.';
import renderTheme from '../../utils/RenderTheme';

describe('<Footer />', () => {
  it('should render', () => {
    renderTheme(<Footer html="<h1>Olá</h1>" />);

    expect(screen.getByRole('heading', { name: 'Olá' })).toMatchSnapshot();
  });
  it('should render a snapShot', () => {
    const { container } = renderTheme(<Footer html="<h1>Olá</h1>" />);

    expect(container).toMatchSnapshot();
  });
});
