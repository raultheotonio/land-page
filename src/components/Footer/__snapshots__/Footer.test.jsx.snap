// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render 1`] = `
<h1>
  Olá
</h1>
`;

exports[`<Footer /> should render a snapShot 1`] = `
<div>
  <div
    class="sc-hLBbgP bQheOD"
  >
    <div
      class="sc-gswNZR DuAtq"
    >
      <p
        class="sc-bcXHqe cfnKmM"
      >
        <h1>
          Olá
        </h1>
      </p>
    </div>
  </div>
</div>
`;
